migration/v0.16.x-v0.17.x/upgrade-apps.md

```diff
@@ -0,0 +1,17 @@
+# Upgrade v0.16.x to v0.17.0
+
+1. Checkout the new release: `git checkout v0.17.0`
+
+1. Rename `clusterAdmin.admins` to `clusterAdmin.users` for both `wc-config.yaml` and `sc-config.yaml`
+
```

Comment:
Forgot about this file in my PR (that is now merged), would you care to add this as a separate commit? :smile: 
```suggestion
1. Rename `clusterAdmin.admins` to `clusterAdmin.users` for both `wc-config.yaml` and `sc-config.yaml`

1. Check that `global.clusterDns` in both `wc-config.yaml` and `sc-config.yaml` matches the IP for the `coredns` service in `kube-system` namespace. Old default values did not match defaults in Kubespray.
```